src/js/stores/notifications.js

```diff
@@ -21,6 +22,35 @@ var NotificationsStore = Reflux.createStore({
     }
   },
 
+  isNewNotification: function (response) {
+    var self = this;
+    var playSound = SettingsStore.getSettings().playSound;
+
+    if (!playSound) { return; }
+
+    // Check if notification is already in the store.
+    var isNew = false;
+    _.map(response, function (obj) {
+      if (!_.contains(self._previousNotifications, obj.id)) {
+        isNew = true;
+      }
+    });
+
+    // Play Sound.
+    if (isNew) {
+      if (playSound) {
+        var audio = new Audio('sounds/digi.wav');
```

Comment:
I would load this when the app first loads. :) Perhaps have a store for sounds / notifications.